frontend/src/styles/pages.css

```
@@ -342,6 +365,83 @@
+  from {
+    opacity: 0;
+    transform: translateY(6px);
+  }
+  to {
+    opacity: 1;
+    transform: translateY(0);
+  }
+}
+
+.dropdown-panel-title {
+  font-family: "Gaegu", inherit;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Gaegu" (font-family-name-quotes)

(font-family-name-quotes)